modules/nodes-scheduler-pod-affinity-example.adoc

```diff
@@ -103,12 +103,7 @@ spec:
     image: docker.io/ocpqe/hello-pod
 ----
 
-* The pod *pod-s2* is not scheduled unless there is a node with a pod that has the `security:s2` label. If there is no other pod with that label, the new pod remains in a pending state:
-+
-----
-NAME      READY     STATUS    RESTARTS   AGE       IP        NODE
-pod-s2    0/1       Pending   0          32s       <none>
-----
+* The pod *pod-s2* cannot be scheduled on the same node as `pod-s1`.
```

Comment:
In the example yaml file  the first pod name is `s1`, so here should be `s1`.

Reply:
@zhouying7780 Thank you for catching that. I changed the first pod name to pod-s1.